deploy/02_sources.ts

```diff
@@ -242,13 +245,18 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
     // one hundred million tokens minted per network on testnets
```

Comment:
why not keep it the same amount of tokens as mainnet at 10 billion if it's supposed to be a mirror

Reply:
Line 249 does that for testnet environment

Reply:
why does it say one hundred million tokens minted per network on testnets then?

Reply:
That is the default being set, which would be true for develop and experimental environments, then we add an override specifically for testnet environment so that it mirrors mainnet settings.

Reply:
okay i see